content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -34,16 +34,16 @@ You must run the RDI installer as a privileged user because it installs
 [containerd](https://containerd.io/) and registers services. However, you don't
 need any special privileges to run RDI processes for normal operation.
 
-The [K3s](https://k3s.io/) Kubernetes distribution used internally by RDI has a few
+RDI has a few
 requirements for cloud VMs that you must implement before running the
 RDI installer, or else installation will fail. The following sections
-give full pre-installation instructions for [RHEL](#k3s-rhel) and
-[Ubuntu](#k3s-ubuntu).
+give full pre-installation instructions for [RHEL](#firewall-rhel) and
+[Ubuntu](#firewall-ubuntu).
```

Comment:
This is one sentence, have it in one line although the outcome is the same.

Reply:
@spicy-sauce TBH, we split lines like this routinely throughout the docs. What you're suggesting does sometimes make sense in a readme file or release notes, etc, where people commonly read the raw text but the raw text of our doc sources is never actually seen by users.

Reply:
OK, as I said it will look the same, but if this is the convention you use, sounds good to me.